packages/expo-ads-admob/android/src/main/java/expo/modules/ads/admob/AdMobRewardedVideoAdModule.java

```diff
@@ -112,6 +113,21 @@ public void run() {
 
         mRewardedAd = new RewardedAd(mActivityProvider.getCurrentActivity(), mAdUnitID);
 
+        if( ssvOptions != null && !ssvOptions.isEmpty() ) {
+          // Check if ReadableArguments has a value (see doc on ReadableArguments here https://github.com/expo/expo/blob/59794c1cc56ac8ea110c208ba0711dbd1e044594/packages/%40unimodules/core/android/src/main/java/org/unimodules/core/arguments/ReadableArguments.java#L10)
+          ServerSideVerificationOptions options = new ServerSideVerificationOptions.Builder();
+          String userId = ssvOptions.getString("userId");
+          String customData = ssvOptions.getString("customData");
+          if(userId != null) {
```

Comment:
what happens if we `setServerSideVerificationOptions` and either the `userId` or `customData` fields are missing?

Reply:
I've verified, it just performs all calls normally like the ServerSideVerificationOptions are not provided.